static/js/biomedical/disease/data_processing_utils.ts

```diff
@@ -285,3 +286,34 @@ export function getCompoundDiseaseContraindication(
   processedData.sort((a, b) => (a.drugSource > b.drugSource ? 1 : -1));
   return processedData;
 }
+
+/**
+ * Fetches the common name of the disease of interest
+ * @param data
+ * @returns - string with disease common name
+ */
+export function getDiseaseCommonName(data: GraphNodes): string {
+  let commonName = null;
+  if (!data) {
+    return;
+  }
+  // check for null values
+  if (_.isEmpty(data.nodes) || _.isEmpty(data.nodes[0].neighbors)) {
+    return;
+  }
+  for (const neighbour of data.nodes[0].neighbors) {
+    if (neighbour.property !== "commonName") {
+      continue;
+    }
+    // check for null or non-existent property values
+    if (_.isEmpty(neighbour.nodes)) {
+      continue;
+    }
+    commonName = neighbour.nodes[0].value;
+    // capitalize the first letter of the disease name
+    const formattedDiseaseName =
+      commonName[1].toUpperCase() + commonName.slice(2);
+    // return formatted disease name with double quotes removed
+    return formattedDiseaseName.replaceAll('"', "");
```

Comment:
I might use `trim` here instead, so only completely quoted names are cleared. Could there be instances where we'd want to keep the double quote e.g. `foo "bar"`?

Reply:
That's a great idea! `.trim()` would remove all double quotes from the string and we don't want to keep any double quotes, so this works perfectly well!